packages/ui/src/overview/hooks/useDeadlines.ts

```diff
@@ -33,13 +41,15 @@ export const useDeadlines = (params?: UseDeadlinesParams) => {
 
   const deadlines = useMemo((): Record<DeadlineNamespace, any[]> => {
     const rawElection = data?.electionRounds[0]
-    const election = rawElection && asElection(rawElection)
+    const election = rawElection && [asElection(rawElection)]
```

Comment:
That's a small detail but lets drop the `rawElection` just to make it consistent with the other deadlines
```suggestion
    const election = data?.electionRounds.map(asElection(rawElection)) ?? []
```